src/gsy_e/models/strategy/load_hours.py

```diff
@@ -84,23 +226,10 @@ def __init__(self, avg_power_W, hrs_per_day=None, hrs_of_day=None,
         as per utility's trading rate
         """
         super().__init__()
-        LoadValidator.validate_energy(
-            avg_power_W=avg_power_W, hrs_per_day=hrs_per_day, hrs_of_day=hrs_of_day)
-        self._state = LoadState()
-        self.avg_power_W = avg_power_W
-
-        # consolidated_cycle is KWh energy consumed for the entire year
-        self.daily_energy_required = None
-        # Energy consumed during the day ideally should not exceed daily_energy_required
-        self.energy_per_slot_Wh = None
-
-        # Maps each simulation day to the number of active hours in that day
-        self.hrs_per_day: Dict[int, int] = {}
+        if not hasattr(self, "_energy_params"):
```

Comment:
What's the case in which a strategy doesn't have the `self._energy_params` attribute?

Reply:
I just found this conversation: https://github.com/gridsingularity/gsy-e/pull/1435/files#r870054300

Reply:
Constructing the strategy itself and not its derived classes. For example, if the DefinedLoadStrategy class (which inherits from LoadHoursStrategy) is constructed, then the self._energy_params will be populated in the constructor of DefinedLoadStrategy class and should not be populated also from the constructor of the LoadHoursStrategy class, therefore in this case the self._energy_params will be populated. When constructing directly the LoadHoursStrategy though, the self._energy_params has not been populated by a derived class and needs to be populated from the constructor of the LoadHoursStrategy class.

Reply:
I'm not sure I understand. Isn't this how it normally works?
```py
class LoadHoursStrategy:
    def __init__(self):
        self._energy_params = "parent value"


class DefinedLoadStrategy(LoadHoursStrategy):
    def __init__(self):
        super().__init__()
        self._energy_params = "child value not overwritten"


>>> DefinedLoadStrategy()._energy_params
# "child value not overwritten"
```
It depends on the position of the `super()` call I suppose. Anyway, if you've already checked the possible available options it's all good for me 👍

Reply:
I was under the impression that some of the EnergyParameters classes might not be able to be constructed due to the dummy parameters that we pass to the superclass, but you are right, I just tested it and overwriting the _energy_params member seems to work as expected! Adding this fix and thanks for the suggestion!